bingus-bot/src/index.ts

```diff
@@ -0,0 +1,118 @@
+import {
+  Client,
+  EmbedBuilder,
+  GatewayIntentBits,
+  REST,
+  Routes,
+  SlashCommandBuilder,
+  SlashCommandStringOption,
+} from "discord.js";
+import auth from "../auth.json" assert { type: "json" };
```

Comment:
Importing JSON files is dangerous. They get copied to the output folder which moves them from where they're supposed to be. I would prefer if this would just use `node:fs#readFileSync()` and then `JSON.parse()` with a validator.

Reply:
No, try building it yourself, this is a JavaScript import assert not typescript

Reply:
How do you want to deal with this? You'll have to rebuild the application every time you want to change this (or edit the JSON file in the `dist` folder) :/

Reply:
![image](https://github.com/SlimeVR/Bingus-Search/assets/3698237/f255f0cc-6dec-49cc-930b-4955076b8678)
it just works:tm:

Reply:
JS file has the json assert
![image](https://github.com/SlimeVR/Bingus-Search/assets/3698237/23d716eb-e7f2-4131-92a1-2adc409adf7d)